libcloud/compute/drivers/azure_arm.py

```diff
@@ -997,6 +997,53 @@ def attach_volume(self, node, volume, ex_lun=None,
             })
         return True
 
+    def resize_volume(self, volume, new_size, ex_resource_group):
```

Comment:
This method is not part of the standard (common) compute API, so it should be prefixed with ``ex_`` ( ``ex_resize_volume``).